build/Uno.UI.Build.csproj

```
@@ -482,7 +482,28 @@
 		</PropertyGroup>
 
 		<Exec Command="dotnet tool install --tool-path $(MSBuildThisFileDirectory)\tools Uno.PackageDiff --version 1.1.0-dev.14" IgnoreExitCode="true" />
-		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix) --other=$(PackageNamePrefix).$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(GITVERSION_SemVer).md" />
+		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix) --other=$(PackageNamePrefix).$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(PackageNamePrefix).$(GITVERSION_SemVer).md" />
+		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix).Foldable --other=..\..\..\NugetPackages-Artifacts-generic-UWP\vslatest-generic\$(PackageNamePrefix).Foldable.$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(PackageNamePrefix).Foldable.$(GITVERSION_SemVer).md" />
+		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix).Lottie --other=$(PackageNamePrefix).Lottie.$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(PackageNamePrefix).Lottie.$(GITVERSION_SemVer).md" />
+		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix).Maps --other=..\..\..\NugetPackages-Artifacts-generic-UWP\vslatest-generic\$(PackageNamePrefix).Maps.$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(PackageNamePrefix).Maps.$(GITVERSION_SemVer).md" />
+		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix).MSAL --other=$(PackageNamePrefix).MSAL.$(GITVERSION_SemVer).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.$(PackageNamePrefix).MSAL.$(GITVERSION_SemVer).md" />
```

We should be able to use something like that:

```suggestion
        <ItemGroup>
           <_diffPackage Include="$(PackageNamePrefix).$(GITVERSION_SemVer)" />
            <_diffPackage Include="$(PackageNamePrefix).Foldable.$(GITVERSION_SemVer)" />
        </ItemGroup>

		<Exec Command="$(MSBuildThisFileDirectory)\tools\generatepkgdiff.exe --base=$(PackageNamePrefix) --other=%(_diffPackage .Identity).nupkg --diffignore=PackageDiffIgnore.xml --outfile=$(OutputDir)\ApiDiff.%(_diffPackage .Identity).md" />
```